Python/Fetch_Contributions/README.md

```diff
@@ -0,0 +1,43 @@
+# Fetch contributions
+
+- Script can be executed as a CLI.
+- Takes arguments username and organization.
+- Returns the pull requests made by user to the organization into a Markdown file. 	
+	- Markdown file consists a table having - "Title to PR" , "Link of PR" , "Status(Merged/Closed/Open)"
+
+## Example Markdown File:
```

Comment:
You have already uploaded the md file, no need to repeat the same in the README> Just provide a link to the file here?